include/modules/hyprland/workspaces.hpp

```diff
@@ -26,12 +26,13 @@ namespace waybar::modules::hyprland {
 
 class Workspaces;
 
-class CreateWindow {
+class WorkspaceWindow {
```

Comment:
> Renamed the CreateWindow class to WorkspaceWindow, to make it clearer it isn't a function but a class.

I agree `CreateWindow` could have a different name to differentiate it from a function, but `WorkspaceWindow` makes it seems like it's a window on a workspace, while in reality it's the payload that will be used to push a new window to a workspace. Maybe `WindowCreationPayload` would be a better compromise between the two?

Reply:
I see. What about `WindowData`? WindowCreationPayload seems a bit long to me ;)

Reply:
`WindowCreationData`/`WinCreateData`, then? Just `WindowData` doesn't seem to be quite enough to decipher the class' purpose

Reply:
My only wrench to throw in is that everything related to the data arguments for it are using the 'payload' nomenclature.

Reply:
Valid points, changed it to WindowCreationPayload.